flink-connectors/flink-connector-aws-kinesis-data-streams/src/main/java/org/apache/flink/connector/kinesis/sink/KinesisDataStreamsException.java

```diff
@@ -39,12 +39,12 @@ public KinesisDataStreamsException(final String message, final Throwable cause)
 
         public KinesisDataStreamsFailFastException() {
             super(
-                    "Encountered an exception while persisting records, not retrying due to {failOnError} being set.");
```

Comment:
Instead of duplicating this string, either call `this(null)` or extract to constant